.travis.yml

```diff
@@ -14,4 +14,7 @@ script:
 after_success:
   - yarn run test-integration
   - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-  - ./scripts/ci-deployment/docker_ci.sh
+  - bash ./scripts/ci-deployment/github/build-docker-images.sh
```

Comment:
I think it's best to keep the build script and the deploy script separate. But I'll happily call the `install-rancher` and `generate-secrets` script from `deploy.sh`